tests/kubernetes/test_pod_generator.py

```diff
@@ -713,10 +713,20 @@ def test_reconcile_specs_init_containers(self):
         assert res.init_containers == base_spec.init_containers + client_spec.init_containers
 
     def test_deserialize_model_file(self):
-        path = sys.path[0] + '/tests/kubernetes/pod.yaml'
-        result = PodGenerator.deserialize_model_file(path)
-        sanitized_res = self.k8s_client.sanitize_for_serialization(result)
-        assert sanitized_res == self.deserialize_result
+        with self.assertLogs(level='WARNING') as capture_logs:
+            path = sys.path[0] + '/tests/kubernetes/pod.yaml'
+            result = PodGenerator.deserialize_model_file(path)
+            sanitized_res = self.k8s_client.sanitize_for_serialization(result)
+            assert sanitized_res == self.deserialize_result
+            assert len(capture_logs.records) == 0
```

Comment:
No change is made to this test case itself, other than checking if the code dumped any warning log